From: David Faure <faure@kde.org>
Date: Mon, 27 Jul 2015 11:29:18 +0000
Subject: Use KDESRCDIR to fix the escaping problems with SOURCE_DIR :)
X-Git-Url: http://quickgit.kde.org/?p=analitza.git&a=commitdiff&h=3901d6b2fe6dbf32636d310dee97134d8e6c723f
---
Use KDESRCDIR to fix the escaping problems with SOURCE_DIR :)
---


--- a/analitzaplot/tests/CMakeLists.txt
+++ b/analitzaplot/tests/CMakeLists.txt
@@ -12,7 +12,6 @@
 kde4_add_unit_test(plotsmodeltest plotsmodeltest.cpp)
 target_link_libraries(plotsmodeltest ${testLibs})
 
-add_definitions(-DSOURCE_DIR="\\\"${CMAKE_CURRENT_SOURCE_DIR}\\\"")
 kde4_add_unit_test(plotsdictionarymodeltest plotsdictionarymodeltest.cpp)
 target_link_libraries(plotsdictionarymodeltest ${testLibs})
 

--- a/analitzaplot/tests/plotsdictionarymodeltest.cpp
+++ b/analitzaplot/tests/plotsdictionarymodeltest.cpp
@@ -41,7 +41,7 @@
 {
 	PlotsDictionaryModel m;
 	m.clear(); //we don't want the installed, we want the ones in the source directory
-	QStringList res = KGlobal::dirs()->findAllResources("data", SOURCE_DIR "/../data/plots/*.plots");
+	QStringList res = KGlobal::dirs()->findAllResources("data", KDESRCDIR "/../data/plots/*.plots"); // Qt5: port to QFINDTESTDATA
 	foreach(const QString& f, res) {
 		m.createDictionary(f);
 	}

